app/controllers/applications/build_controller.rb

```diff
@@ -79,6 +79,7 @@ def handle_form_object(params, step)
     if @form.valid?
       status = { status: get_status(step) }
       form_params.delete('application_id')
+      form_params.delete('emergency') if form_params.key?(:emergency)
```

Comment:
This is messed up, I know. But since we haven't broken up the **Aplication** model into smaller models just yet, this has to go in, for the form to be submitted.